js/screens/Profile/styles.js

```diff
@@ -0,0 +1,50 @@
+import { StyleSheet, Dimensions } from 'react-native';
+import { fonts, colors } from '../../config/styles';
+
+const { height, width } = Dimensions.get('window');
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: 'white',
+    height: height,
+  },
+  profileHeader: {
+    alignItems: 'center',
+    borderBottomColor: colors.darkBlue,
+    borderBottomWidth: 0.5,
+    paddingVertical: 20,
+  },
+  linkText: {
+    marginTop: 10,
+    color: colors.darkBlue,
+    fontFamily: fonts.main,
+    fontSize: 24,
+    fontWeight: 'bold',
+  },
+  editText: {
+    marginTop: 5,
+    fontFamily: fonts.main,
+  },
+  icon: {
+    height: 60,
+    width: 60,
+    overflow: 'visible',
```

Comment:
Please use flexbox instead and Dimensions instead of hard-coded height and width.

Reply:
And the image is a placeholder approved by the designers, the image will appear when the user account info is available.